docs/docs/cloud/aws/compliance.md

```diff
@@ -0,0 +1,204 @@
+# AWS Compliance
```

Comment:
This page is not included in mkdocs.yml.

```
INFO     -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - docs/cloud/aws/compliance.md
```

Reply:
And the page is broken.
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/2253692/205057766-792a98fd-35e5-4aa6-8054-56c5c718a3cd.png">

You can build the doc and test it locally with `make mkdocs-serve`.

Reply:
ah sorry thanks I forgot about this - I fixed it.